### PR TITLE
project 1: embedding video and images

### PR DESCRIPTION
**Pull request by mahmoud717**
in this project we:
1. created a youtube page clone using html&css.
2. embedded the main video from the original youtube page.
3. styled the page so that it looks like the original page.
_______________________
note:
in the next projects we will commit more often.

### CODE REVIEW

**Review by expjazz — CHANGES_REQUESTED** [review]
## Hey @mahmoud717 and @Adewunmi97 

#### There are a few problems with the repo. Here is what you need to change:

- The readme file has no content. You can check more here: https://github.com/microverseinc/readme-template

- The pull request does not have a meaningful title and neither a description. Kindly add them for the next review.

- You should commit more often

#### For more info on these subjects, check this link: https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/

### This Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready! :pray:

Good Luck and Happy Coding :+1: :

P.S. If you have any questions about my review or need help, you can message me on slack (@ Expedito Andrade).:smile:

File: index.html [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+          </div>
+          <div class="video-info">
+            <div class="vid-info-start">
+              11,768,672 views • Jul 8, 2020
+            </div>
+
+            <ul class="vid-info-end">
+              <li><i class="fas fa-thumbs-up"></i> 211K</li>
+              <li><i class="fas fa-thumbs-down"></i> 8K</li>
+              <li><i class="fas fa-share"></i> SHARE</li>
```
Comment (HADDADSOHAIB) [inline index.html]: The share should be a button
- [ ] It has a share button ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/embedding_images_and_video/video_share_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/48945837/))

**Review by HADDADSOHAIB — CHANGES_REQUESTED** [review]
### Status: Change required :recycle: 

### Hi @mahmoud717  and @Adewunmi97 

### You did great with the project, good job :+1: . There is some changes that have to be made   to approve the project:
- The recommended video section needs improvement:

| Source Page	| Clone Page |
|:---:|:---:|
|Screenshot from 2020-07-10 10-30-36 | Screenshot from 2020-07-10 10-30-23 |

- check my comments for additional requirements.
- In the project description, there was a link that you should have used to build your project:
image
http://archive.vn/Bss88
there a slight different between your project and that link in the section under the main video.
- [Optional] use  the [`<nav>` tag](http://html5doctor.com/nav-element/) for the major navigation links
- [Optional]  use  the [`<main>` tag](https://www.w3schools.com/tags/tag_main.asp) for the dominant content of the `<body>` of a document

### Once done with the changes, please ask for another code review.

**Review by BrittanyBlake — CHANGES_REQUESTED** [review]
### Status: Changes Required ♻️ 
****
### Hi @mahmoud717  and @Adewunmi97! 👋 👋 
You have done a great job implementing the changes requested by the previous TSE! 👍 You have also done a wonderful job with your project so far! It looks really good! 👏 👏 👏  There is just one issue that needs to be resolved before you can move on: ⬇️ 
****

- [ ] Please make sure to delete commented code. It is bad practice to leave commented code in your projects as it makes it look untidy. If you need to see a previous version of your code you can do so using your GitHub commit history. Please see my comments for instances of commented code in your files. 

- [ ] Please check my comments for additional changes. 

****
### Please make the required changes and resubmit for a new code review once you are ready 🙏

Good Luck and Happy Coding 🍀 👩🏾‍💻

_P.S. If you have any questions about my review or need help, you can message me on slack (@ Brittany Blake)._

File: index.html [hunk 2]
```diff
@@ -0,0 +1,271 @@
...
+
+            </div>
+            <div class="channel-info-end">
+              <button>SUBSCRIBE</button>
+            </div>
+            
+          </div> -->
+          <!-- <div class="vid-description">Juventus missed the chance to move 10 points clear at the top of Serie A as they incredibly let a two-goal lead slip to lose 4-2 at Milan after a remarkable second half on Tuesday | Serie A TIM
+
+            </div> -->
```
Comment (BrittanyBlake) [inline index.html]: Please remove commented code .

File: styles.css [hunk 3]
```diff
@@ -0,0 +1,360 @@
...
+  border: none;
+  border-radius: 2px;
+  margin: 15px 0 0 0;
+  font-weight: 500;
+}
+.vid-description {
+  margin: 10px 170px 0 60px;
+  line-height: 20px;
+  font-size: 14px;
+} */
```
Comment (BrittanyBlake) [inline styles.css]: Please remove commented code .

File: styles.css [hunk 4]
```diff
@@ -0,0 +1,360 @@
...
+  margin-top: 0;
+}
+.buttons {
+  display: flex;
+  justify-content: space-between;
+}
+.buttons-start {
+  display: inline-block;
+  justify-content: start;
+  padding-left: 0px !important;
```
Comment (BrittanyBlake) [inline styles.css]: Try not to use `!important` in your code. If you need to override something you can do so using selector specificity.  You can learn more about that in [this article](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity) and in [this article](https://www.w3schools.com/css/css_specificity.asp)

**Comment by ikraamg** [comment]
### **Status: PROJECT APPROVED**  :1st_place_medal: :1st_place_medal: :1st_place_medal:


Hi @Adewunmi97  and @mahmoud717  :smile:  Congratulations, your project looks amazing and you did an amazing job improving it and making the necessary changes :sunglasses:. I know it's been a long journey and you should be proud of yourself for pushing it through!  

You can merge and submit your project! 

**A little hint that is sometimes useful:** 
When you want to stop your site from breaking when too small or looking odd on big screens, you can apply the following css code on your **html or body** tag of your page:
_min-width: 350px;_ (you can pick your pixels)
_max-width: 1366px;_ (you can pick your pixels)

If you have any questions, don't hesitate to contact me through slack: **_@ikraam_**


_**Happy coding and good luck on the next project**_  :desktop_computer: 

_Project reviewed by Ikraam Ghoor (TSE)_

**Pull request merged**